qa/tasks/mgr/dashboard/test_motd.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=too-many-public-methods
+
+from __future__ import absolute_import
+
+import time
+
+from .helper import DashboardTestCase
+
+
+class MotdTest(DashboardTestCase):
```

Comment:
In order to get it tested by Teuthology it needs to be added to https://github.com/ceph/ceph/blob/master/qa/suites/rados/dashboard/tasks/dashboard.yaml#L37